src/main/java/connectify/ui/EntityListPanel.java

```diff
@@ -121,6 +125,8 @@ private TitledPane createCompanyPane(Company company) {
         if (company.getPersonList().size() == 0) {
             companyPeopleListView.setPlaceholder(new Label("No person in company"));
         }
+        // Set the preferred width of the ListView to match the ScrollPane's width
+        companyPeopleListView.prefWidthProperty().bind(scrollPane.widthProperty());
```

Comment:
Good fixes to allow the people associated to the Company be more visible in the Graphical User Interface!